src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,48 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_NUMBER_ELEMENTS = 10;
+    private K [] keys;
+    private V [] values;
```

Comment:
```suggestion
    private K[] keys;
    private V[] values;
```
agree with @tanya-logatskaya